apps/video/js/video.js

```diff
@@ -12,9 +12,8 @@ const LAYOUT_MODE = {
 
 var dom = {};
 
-var ids = ['thumbnail-list-view', 'thumbnails-bottom', 'thumbnail-list-title',
+var ids = ['thumbnail-views', 'thumbnails-bottom', 'thumbnail-list-title',
```

Comment:
It turns out, the js doesn't use the 'thumbnail-list-view' so we don't need to add 'thumbnail-views'. Also, 'thumbnails-bottom' is not used so it would be nice if you can remove that as well.